src/main/java/Commands/Command.java

```diff
@@ -0,0 +1,278 @@
+package Commands;
+
+import Tasks.Deadline;
+import Tasks.Event;
+import Tasks.Task;
+import Tasks.ToDo;
+
+import Utilities.Storage;
+import Utilities.TaskList;
+import Utilities.Ui;
+
+import Exceptions.YpxmmException;
+
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+
+/**
+ * Enum representing different commands that can be executed by the program.
+ */
+public enum Command {
+
+    /**
+     * Command to list all tasks.
+     */
+    LIST {
+        /**
+         * Executes the command to list all tasks.
+         *
+         * @param tasklist the list of tasks
+         * @param ui the user interface
+         * @param storage the storage utility
+         * @param parsed the parsed command arguments
+         */
```

Comment:
I like that the Javadocs are in the correct form (especially the singular form verb in the opening sentence). Well done!